spec/models/package_manager/go_spec.rb

```diff
@@ -259,16 +275,17 @@
     end
 
     context "with a package that has incompatible versions only" do
-      it "loads the package and it has no versions" do
+      it "loads the package and ingests versions" do
         package = nil
 
-        VCR.use_cassette("go/update/noaa") do
+        VCR.use_cassette("go/update/docker-distribution") do
           expect do
-            package = described_class.update("github.com/cloudfoundry/noaa/errors")
+            # this example doesn't have a valid go.mod, is a "module", and has only incompatible versions
+            package = described_class.update("github.com/docker/distribution")
           end.to change(Project, :count).from(0).to(1)
         end
 
-        expect(package.versions.count).to eq(0)
+        expect(package.versions.count).to eq(53)
```

Comment:
I had to switch from `noaa/errors` to `docker/distribution` here because the former wasn't technically a Go Module. Also noteworthy that we do ingest "incompatible" versions, so this is more explicit about that too now